### PR TITLE
[foreman] Sort foreman_tasks_tasks by started_at asc

### PR DESCRIPTION
**Pull request by pafernanr**
Modify queries sorting by started_at so we can obtain recent tasks, plans, actions and steps

---
Please place an 'X' inside each '[]' to confirm you adhere to our [Contributor Guidelines](https://github.com/sosreport/sos/wiki/Contribution-Guidelines)

- [x] Is the commit message split over multiple lines and hard-wrapped at 72 characters?
- [x] Is the subject and message clear and concise?
- [x] Does the subject start with **[plugin_name]** if submitting a plugin patch or a **[section_name]** if part of the core sosreport code?
- [x] Does the commit contain a **Signed-off-by: First Lastname <email@example.com>**?
- [ ] Are any related Issues or existing PRs [properly referenced](https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) via a Closes (Issue) or Resolved (PR) line?

### CODE REVIEW

File: sos/report/plugins/foreman.py [hunk 1]
```diff
@@ -194,27 +194,32 @@ def collect_foreman_db(self):
             "'%(pass|key|secret)%'"
         )
 
+        dtaskcmd = ('select * from foreman_tasks_tasks where started_at > '
+                    'NOW() - interval %s order by started_at asc'
+                    % quote(days))
+
```
Comment (jcastill) [inline sos/report/plugins/foreman.py:200]: Would it make sense to substitute this with f-strings?

File: sos/report/plugins/foreman.py [hunk 2]
```diff
@@ -194,27 +194,32 @@ def collect_foreman_db(self):
...
         dyncmd = (
             'select dynflow_execution_plans.* from foreman_tasks_tasks join '
             'dynflow_execution_plans on (foreman_tasks_tasks.external_id = '
             'dynflow_execution_plans.uuid::varchar) where foreman_tasks_tasks.'
-            'started_at > NOW() - interval %s' % quote(days)
+            'started_at > NOW() - interval %s order by foreman_tasks_tasks.'
+            'started_at asc' % quote(days)
         )
```
Comment (jcastill) [inline sos/report/plugins/foreman.py:207]: Same here, f-strings as well?

File: sos/report/plugins/foreman.py [hunk 3]
```diff
@@ -194,27 +194,32 @@ def collect_foreman_db(self):
...
 
         dactioncmd = (
              'select dynflow_actions.* from foreman_tasks_tasks join '
              'dynflow_actions on (foreman_tasks_tasks.external_id = '
              'dynflow_actions.execution_plan_uuid::varchar) where '
-             'foreman_tasks_tasks.started_at > NOW() - interval %s'
-             % quote(days)
+             'foreman_tasks_tasks.started_at > NOW() - interval %s '
+             'order by foreman_tasks_tasks.started_at asc' % quote(days)
```
Comment (jcastill) [inline sos/report/plugins/foreman.py:214]: f-strings here?

**Review by jcastill — COMMENTED** [review]
I'm probably nitpicking here but since there's an ongoing effort and PRs to move to f-strings, we may get ahead with this PR

**Comment by pafernanr** [comment]
second commit replaces printf by f-strings

**Comment by jcastill** [comment]
Can you squash both commits and fix the commit message to match DCO?

**Review by TurboTurtle — APPROVED** [review]

**Pull request merged**